caskdata/cdap_cookbook
# Test updates

Update tests to latest current standards

- Use CentOS 6.6, since 6.5 isn't available, anymore.
- Remove minitest-handler, no longer recommended way to test
